src/dom/Draggable.js

```diff
@@ -1,3 +1,12 @@
+import {Evented} from '../core/Events';
+import {touch} from '../core/Browser';
+import {on, off, preventDefault} from '../dom/DomEvent';

```

Comment:
I'm not really convinced on the aliases for that, but I agree it might be confusing. Will take no action on this, maybe I'll change my mind in a later iteration.